tests/test_encode.py

```diff
@@ -30,11 +38,57 @@ def test_count_encode():
                 pd.Timestamp("2016-05-25 13:30:00.050"),
                 pd.Timestamp("2016-05-25 13:30:00.051"),
             ],
+            # AMZN is unkown category for training dataset
             "ticker": ["MSFT", "MSFT", "GOOG", "GOOG", "AMZN", None],
         }
     )
 
+
+def test_count_encode(t_train, t_test):
     step = ml.CountEncode("ticker")
     step.fit_table(t_train, ml.core.Metadata())
     res = step.transform_table(t_test)
     assert res.to_pandas().sort_values(by="time").ticker.to_list() == [4, 4, 2, 2, 0, 0]
+
+
+def test_onehotencode(t_train, t_test):
```

Comment:
```suggestion
def test_one_hot_encode(t_train, t_test):
```
Nit: For case consistency. :)